core/stencil.config.ts

```diff
@@ -211,4 +211,5 @@ export const config: Config = {
   preamble: '(C) Ionic http://ionicframework.com - MIT License',
   globalScript: 'src/global/ionic-global.ts',
   enableCache: true,
+  transformAliasedImportPaths: true,
```

Comment:
I _assume_ y'all would want this set to `true` (since its an opt-in flag). LMK if not

Reply:
Yes please!